File: if_else/secondmax.c
#include<stdio.h>
int main()
{
    int a,b,c;
    printf("enter the number a :");
    scanf("%d",&a);
    printf("enter the number b :");
    scanf("%d",&b);
    printf("enter the number c :");
    scanf("%d",&c);
    int smax;
    if(a>=b && a>=c)
    {
        if(b>=c)
        {
            smax=b;
            printf("second max is %d \n",smax);
        }else{
            smax=c;
            printf("second max is %d \n",smax);
        }
    }else if (b>=c && b>=a)
    {
        if(c>=a)
        {
            smax=c;
            printf("second max is %d \n",smax);
        }else
        {
            smax=a;
            printf("second max is %d\n",smax);
        }
    }else if(c>=a  && c>=b)
    {
        if(a>=b)
        {
            smax=a;
            printf("second max is %d\n",smax);
        }else
        {

            smax=b;
            printf("second max is %d\n",smax);
        }
    }
    return 0;
}

File: JAVASCRIPT/if_else/secondmax.js
let prompt=require('prompt-sync')()
let a=Number (prompt('enter the value of a  '));
let b=Number (prompt('enter the value of b  '));
let c=Number (prompt('enter the value of c  '));
let max,smax;
// if((a>=b && a<=c)  ||(a>=c && a<=b)){
//     smax=a;
// }else if((b>=a && b<=c) || (b>=c && b<=a)){
//     smax=b;
// }else{
//     smax=c
// }
// console.log(smax);

if(a>b){
    max=a;
    smax=b
}else{
    max=b;
    smax=a;
}if(max>c){
    if(smax>c){
        smax=smax;
    }else{
        smax=c;
    }
}else{
    smax=max;
    max=c;
}
console.log(smax);